test/extended/heapster-deployer.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
```

Comment:
@smarterclayton we could just pull from the specific commit on the GitHub raw access.  I'll lock it at a specific commit.